### PR TITLE
Feature lifting line

### PR DESCRIPTION
**Pull request by planes**
New option to run a lifting line instead of a VLM.

Also buried in here is a VLM rewrite!

### CODE REVIEW

File: regression/scripts/lifting_line/lifting_line.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    random.shuffle(Mc)
+    rho = np.linspace(0.3,1.3,test_num)
+    random.shuffle(rho)
+    mu = np.linspace(5*10**-6,20*10**-6,test_num)
+    random.shuffle(mu)
+    T = np.linspace(200,300,test_num)
+    random.shuffle(T)
+    pressure = np.linspace(10**5,10**6,test_num)
+    
+    # Changed after to preserve seed for initial testing
```
Comment (timdmacdo) [inline regression/scripts/lifting_line/lifting_line.py]: table format

File: trunk/SUAVE/Analyses/Aerodynamics/Lifting_Line.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+        conditions.aerodynamics.
+          lift_coefficient_wing              [-] CL (wing specific)
+        inviscid_wings_lift                  [-] CL
+
+        Properties Used:
+        self.surrogates.
+          lift_coefficient                   [-] CL
+          wing_lift_coefficient[wings.*.tag] [-] CL (wing specific)
+        """          
+        """ process vehicle to setup geometry, condititon and settings
```
Comment (timdmacdo) [inline trunk/SUAVE/Analyses/Aerodynamics/Lifting_Line.py]: old comments

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py [hunk 3]
```diff
@@ -64,120 +65,57 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
     orientation = wing.vertical
 
     n  = configuration.number_panels_spanwise
-    nn = configuration.number_panels_chordwise
+    #nn = configuration.number_panels_chordwise
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: remove this

File: trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+def lifting_line(conditions,settings,geometry):
+    """
+
+    Assumptions:
+    subsonic and unswept
+
+    Source:
+    Traub, L. W., Botero, E., Waghela, R., Callahan, R., & Watson, A. (2015). Effect of Taper Ratio at Low Reynolds Number. Journal of Aircraft.
+    
+    Inputs:
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py]: These need to be filled in

File: trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+    # induced alpha
+    alpha_i = np.cumsum(n*A*np.sin(n*A)/np.sin(thetan),axis=1)
+    
+    # Sectional vortex drag
+    Cdv = Cl*alpha_i
+    
+    # Total vortex drag
+    CDv = np.sum(Cdv*AR*etam,axis=1)
+    
+    #############
```
Comment (timdmacdo) [inline trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py]: Why are the #### here? Also it looks like Cdn is not used.

Comment (planes) [reply trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py]: It's a place holder for airfoils later on.

File: trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py (hunk 4 above)
Comment (planes) [reply trunk/SUAVE/Methods/Aerodynamics/Lifting_Line/Lifting_Line.py]: They have been.

**Pull request merged**